File: P3_aplikacija_Zilic/P3-projektni-main/P3-android-app/app/src/main/java/ffos/p3/ontologija/Ontologija.java
package ffos.p3.ontologija;

import java.io.Serializable;

public class Ontologija implements Serializable {

    public Ontologija() {
        this.sifra = sifra;
        this.drzava = drzava;
        this.glumac = glumac;
       // this.godina = godina;
        this.nagrada = nagrada;
        this.heroj = heroj;
    }

    private String glumac;
    private String drzava;
    //private int godina;
    private String nagrada;
    private String heroj;

    private int sifra;

    public int getSifra() {
        return sifra;
    }

    public String getGlumac() {
        return glumac;
    }

    public void setGlumac(String glumac) {
        this.glumac = glumac;
    }

    public String getDrzava() {
        return drzava;
    }

    public void setDrzava(String drzava) {
        this.drzava = drzava;
    }

    //public int getGodina() {
        //return godina;
        //}

    //public void setGodina(int godina) {
        //    this.godina = godina;
    //}

    public String getNagrada() {
        return nagrada;
    }

    public void setNagrada(String nagrada) {
        this.nagrada = nagrada;
    }

    public String getHeroj() {
        return heroj;
    }

    public void setHeroj(String heroj) {
        this.heroj = heroj;
    }

    public void setSifra(int sifra) {
        this.sifra = sifra;
    }
}

File: P3_aplikacija_Zilic/P3-projektni-main/P3-ontologija-API/src/Zilic/Ontologija.php
<?php

namespace Zilic;

/**
 * @Entity @Table(name="ontologija")
 **/


class Ontologija 

{
    /** @id @Column(type="integer") @GeneratedValue **/
    protected $sifra;

    /**
    * @Column(type="string")
    */
    private $glumac;

   /**
    * @Column(type="string")
    */
    private $drzava;

    /**
    * @Column(type="integer")
    */
    private $godina;

    /**
    * @Column(type="string")
    */
    private $nagrada;


    /**
    * @Column(type="string")
    */

    private $heroj;

   

 

    
    public function getSifra(){
      return $this->sifra;
    }
  
    public function setSifra($sifra){
      $this->sifra = $sifra;
    }

    
    public function getGlumac(){
      return $this->glumac;
    }
  
    public function setGlumac($glumac){
      $this->glumac = $glumac;
    }
  
    public function getDrzava(){
      return $this->drzava;
    }
  
    public function setDrzava($drzava){
      $this->drzava = $drzava;
    }
  
    public function getGodina(){
      return $this->godina;
    }
  
    public function setGodina($godina){
      $this->godina = $godina;
    }
  
    public function getNagrada(){
      return $this->nagrada;
    }
  
    public function setNagrada($nagrada){
      $this->nagrada = $nagrada;
    }
  
    public function getHeroj(){
      return $this->heroj;
    }
  
    public function setHeroj($heroj){
      $this->heroj = $heroj;
    }

  
   


    public function setPodaci($podaci)
    {
      foreach($podaci as $kljuc => $vrijednost){
        $this->{$kljuc} = $vrijednost;
      }
    }
    
  }

?>